pkg/rke/templates/templates.go

```diff
@@ -223,13 +224,16 @@ func LoadK8sVersionedTemplates() map[string]map[string]string {
 			">=1.23.8-rancher1-1 <1.23.14-rancher1-1":  aciv523,
 			">=1.23.14-rancher1-1 <1.23.15-rancher1-1": aciv5234,
 			">=1.23.15-rancher1-1 <1.23.16-rancher2-2": aciv5235,
-			">=1.23.16-rancher2-2 <1.24.0-rancher0":    aciv5236,
+			">=1.23.16-rancher2-2 <1.23.16-rancher2-3": aciv5236,
+			">=1.23.16-rancher2-3 <1.24.0-rancher0":    aciv5271,
 			">=1.24.0-rancher0 <1.24.8-rancher1-1":     aciv523,
 			">=1.24.8-rancher1-1 <1.24.9-rancher1-1":   aciv5234,
 			">=1.24.9-rancher1-1 <1.24.13-rancher2-1":  aciv5235,
-			">=1.24.13-rancher2-1 <1.25.0-rancher0":    aciv5236,
+			">=1.24.13-rancher2-1 <1.24.13-rancher2-2": aciv5236,
+			">=1.24.13-rancher2-2 <1.25.0-rancher0":    aciv5271,
 			">=1.25.0-rancher0 <1.25.9-rancher2-1":     aciv5235,
-			">=1.25.9-rancher2-1":                      aciv5236,
+			">=1.25.9-rancher2-1 <1.25.9-rancher2-2":   aciv5236,
+			">=1.25.9-rancher2-2":                      aciv5271,
```

Comment:
With this range change for `v1.26.4-rancher2-1`, `aciv5271` template will be used yet images version `5.2.3.6` are referenced in the definition of `v1.26.4-rancher2-1`.
I understand that ACI-CNI is not tested for 1.26, but I think the best option here is to bump versions to 5.2.7.1 for `v1.26.4-rancher2-1` to be consistent. Another rational is it won't make it worse and there is a higher chance of newer version of ACI-CNI plugin working on 1.26.